impeller/renderer/backend/vulkan/formats_vk.h

```diff
@@ -160,4 +160,37 @@ constexpr vk::SampleCountFlagBits ToVKSampleCount(SampleCount sample_count) {
   }
 }
 
+constexpr vk::Filter ToVKSamplerMinMagFilter(MinMagFilter filter) {
+  switch (filter) {
+    case MinMagFilter::kNearest:
+      return vk::Filter::eNearest;
+    case MinMagFilter::kLinear:
+      return vk::Filter::eLinear;
+  }
+}
+
+constexpr vk::SamplerMipmapMode ToVKSamplerMipmapMode(MipFilter filter) {
+  vk::SamplerCreateInfo sampler_info;
+  switch (filter) {
+    case MipFilter::kNearest:
+      return vk::SamplerMipmapMode::eNearest;
+    case MipFilter::kLinear:
+      return vk::SamplerMipmapMode::eLinear;
+    default:
```

Comment:
Instead of a default, explicitly list all cases and put an unreachable at the end. Since we are constantly evolving these APIs, its easier to find whats missing when we do add new enum values (hopefully fewer times now).

Reply:
done